idea/testData/quickfix/createFromUsage/createVariable/property/valOnClassNoClassObject3.after.1.kt

```diff
@@ -0,0 +1,5 @@
+class A {
+    companion object {
+        val foo: Int
```

Comment:
Property has no initializer

Reply:
Other existing test cases also have no initializer. For example:
[privateForMembers.kt.after](https://github.com/t-kameyama/kotlin/blob/837e46a9b88e20044153cfd959d5ff2c1af74154/idea/testData/quickfix/createFromUsage/createVariable/property/privateForMembers.kt.after#L4)
[thisInClass.kt.after](https://github.com/t-kameyama/kotlin/blob/a1e86e8bfaa933b72331fc71b6605e3c1476771e/idea/testData/quickfix/createFromUsage/createVariable/property/thisInClass.kt.after#L5)

Should I add initializer in other existing test cases as well?